api/config/appstudio.redhat.com_enterprisecontractpolicies.yaml

```diff
@@ -165,6 +165,9 @@ spec:
                                 effectiveUntil:
                                   format: date-time
                                   type: string
+                                imageRef:
+                                  pattern: ^[a-fA-F0-9]{64}$
```

Comment:
I think we might want to add ':' so that `sha256:123412341234...` can be used.

Reply:
I know there are other possible digest types, but in reality it's always sha256, right?

Maybe..?

```suggestion
                                  pattern: ^sha256:[a-fA-F0-9]{64}$
```

Reply:
done